scripts/actions/edit.py

```diff
@@ -54,10 +54,17 @@
     old = data[fieldObj.name]
     new = ''
     if args.has_key(field):
-        new = fieldObj.filterEdit(args, args[field])
```

Comment:
Previously this was comparing a frontend value vs a backend value.
This now checks for changes in the frontend correctly and then converts all values to backend values separately below.